.github/actions/publish/action.yml

```diff
@@ -26,15 +26,17 @@ runs:
 
     - name: Package the helm chart
       shell: bash
-      run: make package
+      run: |
+        mkdir -p new_chart
+        helm package -d new_chart .
 
     - name: Generate updated index.yaml
       shell: bash
-      run: helm repo index . --url https://launchdarkly.github.io/ld-relay-helm --merge helm-repo/index.yaml
+      run: helm repo index new_chart --url https://launchdarkly.github.io/ld-relay-helm --merge helm-repo/index.yaml
```

Comment:
Merging with a direct subdirectory prefixes the paths with that subdirectory. Instead, we should package into a completely separate folder and merge with a folder from a separate hierarchy.